extensions/duck-duck-go-search/src/utils/useSearch.tsx

```diff
@@ -41,23 +41,20 @@ export function useSearch() {
       cancelRef.current = new AbortController();
 
       try {
-        setIsLoading(true);
-
         if (searchText) {
+          setIsLoading(true);
           const autoSearchResult = await getAutoSearchResults(searchText, cancelRef.current.signal);
           setAutoResults(autoSearchResult);
         } else {
           setAutoResults([]);
         }
-
-        setIsLoading(false);
       } catch (error) {
-        if (error instanceof AbortError) {
-          return;
-        }
-
-        console.error("Search error", error);
-        showToast(Toast.Style.Failure, "Could not perform search", String(error));
+        //         if (error instanceof AbortError) {
+        //           return;
+        //         }
+        //
+        //         console.error("Search error", error);
+        //         showToast(Toast.Style.Failure, "Could not perform search", String(error));
```

Comment:
By design. Otherwise it shows "Could not perform search - TypeError: Cannot read properties of undefined
(reading 'name')" everytime I do a bang search.

Reply:
Can you add logic to keep the alert when it's a normal search? If you think it's too complicated, can you at least remove the (try/catch).

Also, looks like there is no `setLoading(false)` after searching. I can't try the updated extensions right now, but it seems wrong to not have that line?